docs/apps.md

```diff
@@ -249,3 +249,17 @@ This is an experimental app that uses the voxel map to plan a path to a goal. It
 ```bash
 python -m stretch.app.vlm_planning
 ```
+
+### Open-Vocabulary Mobile Manipulation
+
+This is an experimental app that uses a language model to generate commands for the robot given a natural language input. It is not yet fully functional.
```

Comment:
What about it is not fully functional?

Reply:
I am still not fully confident about grasping. But the behavior is similar to ai_pickup